this optimisation of CLONE_SETTLS is not valid used like this, and musl
clone(3) will EINVAL on this use
--
--- a/src/3rdparty/chromium/sandbox/linux/services/credentials.cc
+++ b/src/3rdparty/chromium/sandbox/linux/services/credentials.cc
@@ -97,7 +97,7 @@
   // glibc performs syscalls by calling a function pointer in TLS, so we do not
   // attempt this optimization.
   // TODO(crbug.com/1247458) Broken in MSan builds after LLVM f1bb30a4956f.
-  clone_flags |= CLONE_VM | CLONE_VFORK | CLONE_SETTLS;
+  clone_flags |= CLONE_VM | CLONE_VFORK;
 
   // PTHREAD_STACK_MIN can be dynamic in glibc2.34+, so it is not possible to
   // zeroify tls_buf assigning { 0 }

